src/lib/RequestParser.js

```diff
@@ -128,15 +128,16 @@ class RequestParser { // eslint-disable-line no-unused-vars
      * @returns {Nimiq.Transaction}
      */
     parseTransaction(object) {
+        if (!object || typeof object !== 'object' || object === null) {
+            throw new Errors.InvalidRequestError('Request must be an object');
```

Comment:
```suggestion
        if (!object || typeof object !== 'object') {
            throw new Errors.InvalidRequestError('Transaction info must be an object');
```

- The `object === null` check is already covered by `!object`.
- The transaction is being parsed here, not the entire request.

Reply:
Resolved in #524